mate-base/caja/caja-1.26.3.ebuild

```diff
@@ -0,0 +1,92 @@
+# Copyright 1999-2023 Gentoo Authors
```

Comment:
In the `pkgdiff-mg -b ....` output for caja, I see:
```
[...]
+m4_define(gtk_layer_shell_minver,      0.8.0)
[...]
```

But I don't see a gtk-layer-shell dependency in the ebuild.

There is also a new Wayland configure option:
```
+AC_ARG_ENABLE(wayland,
+              [AS_HELP_STRING([--enable-wayland],
+                              [Explicitly enable or disable Wayland support
+                              (default is to enable only if Wayland client development library is detected)])],
+              [enable_wayland=$enableval],
+              [enable_wayland=auto])
+
+# Check if we have gtk-layer-shell installed, and thus should build with Wayland support
+have_wayland=no
+m4_define([WAYLAND_DEPS], [gtk-layer-shell-0 >= gtk_layer_shell_minver wayland-client])
+AS_IF([test "x$enable_wayland" = "xyes"],
+      [PKG_CHECK_MODULES([WAYLAND], [WAYLAND_DEPS], [have_wayland=yes])],
+      [test "x$enable_wayland" != "xno"],
+      [PKG_CHECK_MODULES([WAYLAND], [WAYLAND_DEPS], [have_wayland=yes], [have_wayland=no])])
+
+AM_CONDITIONAL(ENABLE_WAYLAND, [test "x$have_wayland" = "xyes"])
+
+if test "x$have_wayland" = "xyes"; then
+  AC_DEFINE(HAVE_WAYLAND, 1, [Have the Wayland development library])
+fi
```
... that's not wired up either yet.

Reply:
Please make sure there's not other stuff like this missing from other pkgs.

Reply:
I left it out of 1.26.x builds but is enabled in all 1.27 which I found.

Reply:
Oh, I didn't spot you were doing 1.26.x as well, ok

Reply:
I did a few with important bug fixes which may help, I can drop these and just push 1.27 if that makes more sense.

Reply:
nah, it's ok, I just didn't cotton on